src/gmtconnect.c

```diff
@@ -586,43 +586,43 @@ int GMT_gmtconnect (void *V_API, int mode, void *args) {
 				dd[SEG_I][END_A] = dd[SEG_J][END_B] = DBL_MAX;	/* Flag as single line segment so two ends are not used to connect them */
 				/* Only compute a distance for this single segment if it has more than 2 points.  2 point cannot connect back on itself as it would make a polygon with zero area */
 				dd[SEG_I][END_B] = dd[SEG_J][END_A] = (segment[iseg].n < 3) ? DBL_MAX : gmt_distance (GMT, segment[iseg].x_end[END_A], segment[iseg].y_end[END_A], segment[iseg].x_end[END_B], segment[iseg].y_end[END_B]);
-    				nearest_end[SEG_I][END_A] = nearest_end[SEG_J][END_A] = END_B;	/* Duplicate the nearest ID info since it is a single line segment compared to itself */
```

Comment:
The indentations don't look right.

Reply:
Seems like Xcode editor likes to add spaces not tabs.  Need to see if there is a setting for this.  Also, would be nice if this was automated so that indentations were always one tab per level.